<ul><li>Assembly Sequences (contigs, scaffolds, super scaffolds/chromosomes)&nbsp;</li><li>Assembly types (Genome, Transcriptome, Transcripts)</li><li>Raw sequence data used for generating the assembly</li><li>Features/Annotations</li><li>Coordinate translations&nbsp;(where available/existing)</li><li>Patches (where available/existing)</li><li>Alignments (where available/existing)</li><li>Indices (bwa, samtools, ...)</li><li>Registry / meta data</li><li>Release history? (assembly_build and release date)</li></ul><h1 data-label="682002" class="ltx_title_section"></h1><h1 data-label="682002" class="ltx_title_section">Data Files</h1><div></div><div>The data for an assembly will be stored&nbsp;in the following data file formats:</div><div></div><ul><li>FASTA contigs, scaffolds, superscaffolds/chromosome (compressed)</li><li>FASTQ Read files RAW data (compressed)</li><li>BAM alignment and corresponding index file</li><li>AGP Coordinate system translation</li><li>GFF3 Feature Format File: features, liftover</li><li>YAML Data registry metadata file&nbsp;<a href="https://github.com/PlantandFoodResearch/data-registry" target="_blank">data-registry doc</a></li><li>TAB patches</li><li>Common indices&nbsp; (bwa, bowtie2, blast, ...)</li><li>Patch files, patch scripts&nbsp;</li><li>BSGenome packages - see a&nbsp;<a href="https://github.com/PlantandFoodResearch/NGSUserGroup/blob/master/perl/scripts/genome/bsgenome_forge_from_ensdb.pl" target="_blank">fine forge-script</a></li></ul><div></div><div>The minimum req</div><div></div><h2 data-label="972848" class="ltx_title_subsection">Validation</h2><div>Data files stored with the assembly will be validated against&nbsp;file standards using the following validators:</div>