<h1 data-label="682002" class="ltx_title_section">Description</h1><div>The goal of this document is to provide what information is required to make a PFR generated genome assembly fit for purpose to ensure that the downstream analysts are able to easily understand the source and meaning of the files that are available to them. The principal aim is&nbsp;<b>enhanced collaboration, publication&nbsp;</b>and<b> error minimisation</b>&nbsp;through this improved documentation and specification.</div><h3 data-label="814051" class="ltx_title_subsubsection"></h3><h3 data-label="814051" class="ltx_title_subsubsection">What does"fit for purpose"?</h3><div>What is "fit for purpose" mean in this document? A downstream analyst, scientist or reviewer must be able to use the data that is presented in a reasonable timeframe and with an acceptable error model. Errors often creep in from misunderstandings which are greatly minimised if all parties know the expectations.&nbsp;</div><div></div><h1 data-label="682002" class="ltx_title_section">What does Good Look Like</h1><ul><li>Release process automated</li><li>Standards followed</li><li>Issues can be raised and captured to enable a reiterative, collaborative process</li><li>Additional effort not required&nbsp; for publication</li></ul><div></div><h1 data-label="682002" class="ltx_title_section">Roles and Responsibilities</h1><div></div><ul><li>SGL: Approval</li><li>Assembly Steward / Gatekeeper: Quality assurance</li><li>Bioinformatician: Data generation&nbsp;</li></ul><h1 data-label="682002" class="ltx_title_section">Publication Levels</h1><div>A release is the publication of a genome assembly for analytical use. Any publication that uses a genome assembly must refer to a specific release version. Releases may have different degrees of public access. These are defined as:</div><div></div><ul><li>PFR private (e.g. customer relationship)&nbsp;</li><li>PFR intern (e.g. pre-releases or species that don't have publication approval)</li><li>Open access</li></ul><div></div><h1 data-label="682002" class="ltx_title_section">Naming Convention for a Release</h1><div></div><div>Directories to store data in</div><div></div><div>Breeding plants:&nbsp; </div><ul><li>Full name: &lt;species[_cultivar]&gt;/&lt;assembly_build&gt;</li></ul><div>Bacteria:&nbsp;</div><ul><li>Full name: &lt;species&gt;_&lt;strand&gt;/&lt;assembly_build&gt;</li></ul><div>Fungi:</div><ul><li>Full name: &lt;species&gt;_&lt;strand&gt;/&lt;assembly_build</li></ul><div>Vertebrate:</div><div>â€¢&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Full name: &lt;species&gt;_&lt;population of origin&gt;/&lt;assembly_build&gt;</div><div>            </div><h1 data-label="128703" class="ltx_title_section">Pre-Releases</h1><div>Pre-releases follow&nbsp;this standard. The assembly build tag for a pre-release is 'pre'&nbsp; and can contain a version number (e.g. Actinidia_chinensis_Russel_v2.1_pre).&nbsp; A pre-release can contain a subset of a full release data file set.</div><div></div><h1 data-label="357661" class="ltx_title_section">Release Platforms</h1><ul><li>Ensembl (local/public for genome representation and visualisation/analyses)</li><li>Webapollo (for user feedback on gene annotations)</li><li>Directory structure on powerPlant (/output for assembly and /input for raw data used for the assembly)</li><li>In-house BLAST server</li><li>Gene models in BioView databases</li><li>IGV (optional)</li><li>Geneious archive&nbsp; (optional)&nbsp;</li></ul><h1>Genome Assembly Data</h1><div>A release will comprise the following data:</div><ul><div></div><li>Assembly Sequences (contigs, scaffolds, super scaffolds/chromosomes)&nbsp;</li><li>Assembly types (Genome, Transcriptome, Transcripts)</li><li>Link to raw sequence data used for generating the assembly</li><li>Features/Annotations</li><li>Coordinate translations&nbsp;(where available/existing)</li><li>Patches (where available/existing)</li><li>Alignments (where available/existing)</li><li>Indices (bwa, samtools, ...)</li><li>Registry / meta data including the naming of the assembly according to the terminology published by Chain et al in <a href="http://science.sciencemag.org/content/326/5950/236.full" target="_blank">Genome Project Standards in a New Era of Sequencing</a></li><li>Release history? (assembly_build and release date)</li><li>Metrics summaries as files in output as well as in Git&nbsp; (Metrics from Assemblathon2's&nbsp;<a href="https://github.com/ucdavis-bioinformatics/assemblathon2-analysis/blob/master/assemblathon_stats.pl)" target="_blank">assemblaton_stats.pl</a> in preference to faStats from cndsrc, assessment of correctness and completeness - DNASeq mapback rates, RNASeq mapback rates, BUSCO; Accuracy check with <a href="http://www.sanger.ac.uk/science/tools/reapr" target="_blank">REAPR</a>&nbsp;or maybe also Mark Fiers' Hagfish boxplots of largest MPE library for chromosomes; Contamination check using <a href="https://ccb.jhu.edu/software/kraken/" target="_blank">Kraken</a>, <a href="http://kaiju.binf.ku.dk/" target="_blank">Kaiju</a>&nbsp;or <a href="https://sourceforge.net/projects/tuit/" target="_blank">TUIT</a>)</li></ul><h1 data-label="682002" class="ltx_title_section"></h1><h1 data-label="682002" class="ltx_title_section">Data Files</h1><div></div><div>The data for an assembly will be stored&nbsp;in the following data file formats:</div><div></div><ul><li>FASTA contigs, scaffolds, superscaffolds/chromosome (bgzip compressed)</li><li>FASTQ Read files RAW data (bgzip compressed, ai)</li><li>BAM alignment and corresponding index file</li><li>AGP Coordinate system translation</li><li>GFF3 Feature Format File: features, liftovers</li><li>YAML Data registry metadata file&nbsp;<a href="https://github.com/PlantandFoodResearch/data-registry" target="_blank">data-registry doc</a></li><li>TAB patches</li><li>Common indices&nbsp; (bwa, bowtie2, blast, samtools)</li><li>Patch files, patch scripts&nbsp;</li><li>BSGenome packages - see a&nbsp;<a href="https://github.com/PlantandFoodResearch/NGSUserGroup/blob/master/perl/scripts/genome/bsgenome_forge_from_ensdb.pl" target="_blank">fine forge-script</a></li></ul><div></div><div>The minimum requirement for data files is:</div>